cdci_osa_plugin/osa_dataserve_dispatcher.py

```diff
@@ -686,6 +686,9 @@ def check_query_roles(self, provided_roles: List[str], par_dic: dict):
         if max_pointings > 50 or len(scw_list.split(",")) > 50:
             needed_roles.append('unige-hpc-full')            
 
+        if max_pointings > 500 or len(scw_list.split(",")) > 500:
+            needed_roles.append('unige-hpc-extreme') 
```

Comment:
I guess the table with the various roles should be updated. And ideally also some tests

Reply:
You mean for this new role? No let's not deal with that now. Let's assume it is not implemented. We discuss on Monday.

Reply:
I added [there](https://github.com/oda-hub/doc-multi-user/blob/main/plan-roles-users.md) to keep track a bit better.